Controller/EnterpriseAjaxOptionController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Pim\Bundle\CustomEntityBundle\Controller;
+
+use Pim\Bundle\UIBundle\Controller\AjaxOptionController as BaseAjaxOptionController;
+use Pim\Component\ReferenceData\Repository\ReferenceDataRepositoryInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Overridden because the data locale is not taken into account in the AjaxOptionController
+ *
+ * @author Romain Monceau <romain@akeneo.com>
```

Comment:
Really?

Reply:
yes, the only change with the CE version is line 5